devicetypes/aamann/enlighten-envoy-local.src/enlighten-envoy-local.groovy

```diff
@@ -296,9 +302,10 @@ def ping() {
 
 def startPoll() {
 	unschedule()
-	// Schedule 2 minute polling
+	def wantMin = 6       // Schedule N minute polling Integer < 60
```

Comment:
2 mins seemed like a lot...
check interval is normally 2x your poll, but yes, you could parameterize it

setting for poll is ok.   I don't think you need less than the update interval...this is really a comment for what do you choose for the default.